api/models/trigger.go

```diff
@@ -71,46 +77,63 @@ func ValidTriggerType(a string) bool {
 }
 
 var (
+	//ErrTriggerIDProvided indicates that a trigger ID was specified when it shouldn't have been
```

Comment:
ouch, this looks painful!

Reply:
https://github.com/golang/lint/issues/263 
It's the style guide